src/parser/core/modules/BuffWindow.tsx

```diff
@@ -302,6 +302,25 @@ export abstract class BuffWindowModule extends Module {
 		return undefined
 	}
 
+	private countMissedTrackedActions(buffWindow: BuffWindowState, action: BuffWindowTrackedAction): number {
+		const expected = this.getBuffWindowExpectedTrackedActions(buffWindow, action)
+		const actual = buffWindow.getActionCountByIds([action.action.id])
+		const comparator = this.changeComparisonClassLogic(buffWindow, action)
+
+		// If a custom comparator is defined for this action, and it didn't return negative, don't count this window
+		if ( comparator && !comparator(actual, expected).negative ) { return 0 }
+
+		// If this buff window was rushed, again don't count it
+		if ( this.buffStatus.duration ) {
```

Comment:
Should this be accounted for in getBuffWindowExpectedTrackedActions instead, similar to what we do for expected GCDs?  Probably either through a reduceExpectedTrackedActionEndOfFight overrideable function, since it may vary from class to class when/whether they want to "forgive" expected actions during a rushing end of fight window.